.changes/unreleased/Fixes-20241030-201214.yaml

```diff
@@ -0,0 +1,6 @@
+kind: Fixes
+body: Fix `SourceScanOptimizer` generated duplicated columns when querying for 2 derived metrics which define the same alias
```

Comment:
Would change this wording slightly since `SourceScanOptimizer` isn't actually _generating_ the columns. Something like:
> Prevent `SourceScanOptimizer` from combining nodes that use the same input metric alias in different derived metrics